gateway-ha/src/main/java/io/trino/gateway/ha/router/QueryCountBasedRouter.java

```diff
@@ -32,7 +31,8 @@
 public class QueryCountBasedRouter
         extends StochasticRoutingManager
 {
-    private static final Logger log = LoggerFactory.getLogger(QueryCountBasedRouter.class);
+    private static final Logger log = Logger.get(HaGatewayManager.class);
```

Comment:
The argument should be `QueryCountBasedRouter.class`

Reply:
oops good catch